hexa/plugins/connector_dhis2/datacards.py

```diff
@@ -25,6 +26,29 @@ class InstanceSection(Section):
     url = URLProperty(url="url")
 
 
+class UsageSection(Section):
+    title = "Code samples"
+
+    usage_python = CodeProperty(
+        label="Usage in Python", code="get_python_usage", language="python"
+    )
+
+    def get_python_usage(self, item: Instance):
+        return """
+%pip install --upgrade git+https://@github.com/blsq/blsq_dqapp.git
```

Comment:
`blsq_dqapp` is too specific to our use cases at Bluesquare. 

Let's rewrite this section using `git@github.com:davidhuser/dhis2.py.git`

```bash
pip install dhis2.py
```

```python
from dhis2 import Api

api = Api('play.dhis2.org/demo', 'admin', 'district')
```